examples/terraform-azure-example/README.md

```diff
@@ -41,5 +41,90 @@ it should be free, but you are completely responsible for all Azure charges.
 1. Install [Terraform](https://www.terraform.io/) and make sure it's on your `PATH`.
 1. Install [Golang](https://golang.org/) and make sure this code is checked out into your `GOPATH`.
 1. `cd test`
-1. `dep ensure`
+1. `go mod init github.com/<yourrepo>/terratest-module`
+1. `go build terraform_azure_example_test.go`
+1. Make sure [the azure-sdk-for-go versions match](#check-go-dependencies) in the go.mod file and in the test Go file.
+1. [Check environment variables](#check-environment-variables).
 1. `go test -v -run TestTerraformAzureExample`
+
+
+
+
+## Check Go Dependencies
+
+Check that the /azure-sdk-for-go version must match the version in the terratest mod.go file.  
+
+> This was tested with **go1.14.1**.  We have included a sample **go.mod** to correspond with the terraform_azure_Example_test.go test, but these steps will include details on how to generate the go module and include matching dependencies.
```

Comment:
I think adding a sample go.mod file will be confusing long term as the go.mod file will change frequently over time and this sample is likely not going to stay up to date.

Reply:
Sure, will point to the existing one then.